HabitRPG/Views/UserTopHeader.swift

```diff
@@ -65,8 +65,8 @@ class UserTopHeader: UIView, Themeable {
         
         gemView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(showGemView)))
         
-        usernameLabel.font = CustomFontMetrics.scaledSystemFont(ofSize: 16)
-        levelLabel.font = CustomFontMetrics.scaledSystemFont(ofSize: 11)
```

Comment:
So I'm not entirely sure why this is not working correctly, but the CustomFontMetrics font should already be scaling the text. If you check other usages, it does work there. We have this class, so that we can have more control over the font configuration than we would with just the built in textStyles, so this should ideally stay the same.

Reply:
Thanks for your reply! 😎

Sorry, I didn't investigate properly:
What actually happens (at least on my iPhone SE iOS 12.3.1) is that if I just used Habitica app, then go to Settings, change Dynamic Type & then go back to Habitica, usernameLabel & levelLabel stays the wrong size.
If I subsequently force-quit Habitica & reopen it, it's suddenly the right size 🤔


Should I open an issue & investigate further?
(IMO it probably isn't that big of a deal, because I guess most people don't change their text size frequently enough)